src/components/Navbar/Navbar.types.ts

```diff
@@ -1,9 +1,9 @@
-import { NavbarProps } from 'decentraland-ui'
+import { NavbarProps } from 'decentraland-ui/dist/components/Navbar/Navbar.types'
 import { Dispatch } from 'redux'
 
-export type Props = Partial<NavbarProps> & { isNewNavbarEnabled: boolean; isNavbarV2Enabled: boolean; hasPendingTransactions: boolean }
+export type Props = Partial<NavbarProps> & { isNewNavbarEnabled: boolean; hasPendingTransactions: boolean }
 
-export type MapStateProps = Pick<Props, 'hasPendingTransactions' | 'isConnected' | 'isNewNavbarEnabled' | 'isNavbarV2Enabled'>
-export type MapDispatchProps = Pick<Props, 'onSignIn'>
+export type MapStateProps = Pick<Props, 'hasPendingTransactions' | 'isSignedIn' | 'isNewNavbarEnabled'>
+export type MapDispatchProps = Pick<Props, 'onClickSignIn'>
```

Comment:
got this error when trying to sign in
<img width="513" alt="image" src="https://github.com/decentraland/builder/assets/3170051/2e1e90b0-18ca-4f60-96b8-efff60cc242e">

in the `decentraland-dapps` changes, the property is declared as `onSignIn ` https://github.com/decentraland/decentraland-dapps/pull/565/files#diff-bb66b47d0e404d14c399a2fd3f0561becdc6360289ca7ef4a238accc93c7bf73R22

Reply:
yep, i was taking props from decentraland-ui instead of decentraland-dapps. thx